server/game/cards/02_SHD/units/GreySquadronYWing.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer } from '../../../core/Constants';
+
+export default class GreySquadronYWing extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '9472541076',
+            internalName: 'grey-squadron-ywing',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addOnAttackAbility({
+            title: 'An opponent chooses a unit or base they control. You may deal 2 damage to it',
```

Comment:
This ability needs `optional: true`, and need to add a test confirming that the pass button is present

Reply:
Nice one, didn't read the card that well - indeed it is `You may`